### ISSUE TITLE
Version string

### ISSUE THREAD

**Issue by SlimeDog** [bensku/Skript#943]
Version 2.2-dev32

Internally:
```
version Skript
[11:40:21 INFO]: Skript version 2.2-dev31c
[11:40:21 INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[11:40:21 INFO]: Website: https://bensku.github.io/Skript
[11:40:21 INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, tim740, Tuke-Nuke, TheLimeGlass, xXAndrew28Xx, Sashie, RoyCurtis, nfell2009, Syst3ms and Snow-Pyon
```

**Comment by bensku** [comment]
Download it again. I fixed this a few hours after release.
